go/cmd/zk/zkcmd.go

```diff
@@ -500,6 +503,15 @@ func cmdRm(ctx context.Context, subFlags *flag.FlagSet, args []string) error {
 	return nil
 }
 
+func cmdAddAuth(ctx context.Context, subFlags *flag.FlagSet, args []string) error {
+	subFlags.Parse(args)
+	if subFlags.NArg() < 2 {
+		return fmt.Errorf("addAuth: expected args <scheme> <auth>")
+	}
+	scheme, auth := subFlags.Arg(0), subFlags.Arg(1)
+	return zconn.AddAuth(ctx, scheme, []byte(auth))
```

Comment:
are there any assumptions about the args that should be validated here?

Reply:
I think we can only validate if we limit the schemes -- e.g., `digest` scheme should have format `user:pass`, `ip` scheme should have format `addr/mask`. However, ZooKeeper's auth system is pluggable, so someone could write scheme `foobar` with some unknown format which we cannot validate.